organization/views.py

```diff
@@ -1,5 +1,32 @@
-# -*- coding: utf-8 -*-
-from __future__ import unicode_literals
+from django.contrib.auth.decorators import login_required
+from django.shortcuts import render, HttpResponse
+from organization.models import InternalGroup
+from rest_framework import status
 
 
-# Create your views here.
+@login_required
+def internal_group(request, internal_group_id):
```

Comment:
should this be renamed `internal_group_detail`? Feels like its a bit too close to the naming when we make a general landing page which would probably be named `internal_groups`.